src/ducks/contents.js

```diff
@@ -26,12 +26,12 @@ const projectContentsReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchProjectContents = () => {
+const fetchProjectContents = (project_id) => {
```

Comment:
You're requesting `project_contents` with `project_id`, which isn't right. `project_contents` id is needed, which available in project `links`.